installer/pkg/components/proxy/templates/configmap/vhost.minio.tpl

```diff
@@ -3,7 +3,17 @@ https://minio.{{.Domain}} {
     import remove_server_header
     import ssl_configuration
 
-    reverse_proxy {{.ReverseProxy}} {
+    reverse_proxy {{.ConsoleReverseProxy}} {
+        flush_interval -1
+    }
+}
+
+https://minio-api.{{.Domain}} {
+    import enable_log
+    import remove_server_header
+    import ssl_configuration
+
+    reverse_proxy {{.APIReverseProxy}} {
         flush_interval -1
     }
 }
```

Comment:
What's the motivation for making the MinIO API endpoint globally accessible?

Reply:
Truthfully, I've no idea. It was Past Simon and I'm not responsible for him

I think what I was getting at was that the Bitnami chart exposes the console and the API separately. Of course, we don't need to publicly expose the API, just connect everything to that internally.